python/lsst/ts/cRIOpy/GUI/TimeChart.py

```diff
@@ -105,18 +111,26 @@ def __init__(self, items, maxItems=50 * 30, updateInterval=0.1):
         self.updateInterval = updateInterval
         self.updateTask = make_done_future()
 
-        self._caches = []
-        for axis in items.items():
-            data = [("timestamp", "f8")]
-            for d in axis[1]:
-                if d is None:
-                    self._caches.append(TimeCache(maxItems, data))
-                    data = [("timestamp", "f8")]
-                else:
-                    data.append((d, "f8"))
-                    self._addSerie(d, axis[0])
-            self._caches.append(TimeCache(maxItems, data))
+        self._createCaches(items, maxItems)
+
+        self._attachSeries()
+
+    def _addSerie(self, name, axis):
+        s = QtCharts.QLineSeries()
+        s.setName(name)
+        # s.setUseOpenGL(True)
```

Comment:
Do you still need this commented code?

Reply:
It wasn't working at some stage. Now it seems to be happy and looks like graphing is much faster - uncommented it.